modules/installation-special-config-rtkernel.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-customizing.adoc
+
+[id="installation-special-config-rtkernel_{context}"]
+
+= Adding a real-time kernel to nodes (during installation)
+
+Some {product-title} workloads require a high degree of determinism.
+While Linux is not a real-time operating system, the Linux real-time
+kernel includes a preemptive scheduler that provides the operating
+system with real-time characteristics.
+
+If your {product-title} workloads require these real-time characteristics,
+you can set up your compute (worker) and/or master machines to use the
+Linux real-time kernel when you first install the cluster. To do this,
+create a MachineConfig object and inject that object into the set of manifest
+files used by Ignition during cluster setup, as described in the following
+procedure.
+
+[NOTE]
+====
+This procedure is fully supported with bare metal installations using
+systems that are certified for Red Hat Enterprise Linux for Real Time 8.
+Real time support in {product-title} is also limited to specific subscriptions.
+====
+
+.Prerequisites
+* For a bare metal installation of {product-title}, prepare masters and workers.
+* Use {product-title} version 4.4 or later.
+
+.Procedure
+
+. Generate the Kubernetes manifests for the cluster:
++
+----
+$ ./openshift-install create manifests --dir=<installation_directory>
+----
+
+. Decide if you want to add the real-time kernel to worker or master nodes.
+
+. In the `openshift` directory, create a file (for example,
+`01-worker-realtime.yaml.yaml`) to define a MachineConfig
```

Comment:
The file name 01-worker-realtime.yaml.yaml has 2 suffix. I suggest to change to 01-worker-realtime.yaml